contents/docs/self-host/configure/upgrading-posthog.md

```diff
@@ -4,19 +4,18 @@ sidebar: Docs
 showTitle: true
 ---
 
-We want to make sure upgrading PostHog is as smooth as possible. We use Django's standard migrations to make sure any database migrations are atomic, easy to roll back and generally safe to run.
+We want to make sure upgrading PostHog is as smooth as possible. We use a synchronous and an asynchronous migration framework to make sure database migrations are easy to be deployed, rolled back and safe to run.
 
-Because some data migrations require touching the Events table which can get very large for some instances, some migrations are a little more involved. We will explicitly mention this in the Changelog. We aim to minimize these types of migrations as much as possible, and their frequency will decrease over time. Every time, we will make sure there is a management command you can run in parallel, so you can minimise the amount of downtime to seconds.
+Because some data migrations require touching tables with that can get very large for some PostHog installation, some migrations are a little more involved. We will explicitly mention this in our changelog. We aim to minimize these types of migrations as much as possible, and their frequency will decrease.
```

Comment:
```suggestion
Because some data migrations require touching tables that can get very large for some PostHog installation, some migrations are a little more involved. We will explicitly mention this in our changelog. We aim to minimize these types of migrations as much as possible, and their frequency will decrease.
```